extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -486,6 +494,78 @@ export class ProjectsController {
 		}
 	}
 
+	public async schemaCompareGetTargetScripts(projectFilePath: string): Promise<string[]> {
+		const project = await Project.openProject(projectFilePath);
+		let files: string[] = [];
+
+		const fileEntries: FileProjectEntry[] = project.files;
+
+		fileEntries.forEach(f => {
+
+			if (f.fsUri.fsPath.endsWith('.sql')) {
+				files.push(f.fsUri.fsPath);
+			}
+		});
+
+		return files;
+	}
+
+	public async schemaCompareGetDsp(projectFilePath: string): Promise<string> {
```

Comment:
this doesn't look like it's schema compare specific. Maybe just rename it to getProjectDsp?